File: abc/abc213/b.cpp
// https://atcoder.jp/contests/abc213/submissions/26748124

#include <bits/stdc++.h>
using namespace std;
int main() {
    int n, num; cin >> n;
    vector<pair<int,int>>v;
    for(int i = 1; i <= n; i++) {
      cin >> num;
      v.push_back({num, i});
    }
    sort(v.begin(),v.end());
    cout << v[n-2].second;
}

File: abc/abc213/b.java
// https://atcoder.jp/contests/abc213/submissions/43793384

import java.util.Scanner;
import java.util.Arrays;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int[] a = new int[n];
    for(int i = 0; i < n; i++) a[i] = stdIn.nextInt();
    
    int maxIdx = 0;
    for(int i = 1; i < n; i++) {
      if(a[maxIdx] < a[i]) {
        maxIdx = i;
      }
    }

    int ignore = maxIdx;
    int max = 0;
    int idx = 0;

    for(int i = 0; i < n; i++) {
      if(max < a[i] && i != ignore) {
        max = a[i];
        idx = i;
      }
    }

    System.out.println(idx + 1);
  }
}
